CAIPs/caip-294.md

```diff
@@ -173,20 +178,20 @@ const walletData = {
 }
 ```
 
-This `walletData` type is is a superset of `WalletAnnounceRequestParams` type described in the [CAIP-282][caip-282] standard, adding the optional `extensionId` property as it is only relevant for browser extension based wallets.
+This `walletData` type is is a superset of `WalletAnnounceRequestParams` type described in the [CAIP-282][caip-282] standard, adding the optional `target` property with the `extensionId` as it is only relevant for browser extension based wallets.
 
-### ExtensionId
+### Target
 
-When the `extensionId` is included in the `walletData` object, it indicates that the wallet supports communication via the browser's `externally_connectable` API. In this case:
+When the `target` with [CAIP-341](https://github.com/ChainAgnostic/CAIPs/blob/656551f800843b92243fb08ca6c24e805ad149a3/CAIPs/caip-341.md) type is included in the `walletData` object, it indicates that the wallet supports communication via the browser's `externally_connectable` API. In this case:
 
-1. The dapp MUST use the `extensionId` to establish a connection with the wallet using the `externally_connectable` browser API.
+1. The dapp MUST use the `target.value` (represents `extensionId`) to establish a connection with the wallet using the `externally_connectable` browser API.
```

Comment:
```suggestion
1. The dapp MUST use the `target.value` (an [`extensionId`][externally_connectable]) to establish a connection with the wallet using the `externally_connectable` browser API.
```

Reply:
resolved in [c06811b](https://github.com/pedrouid/CAIPs/pull/9/commits/c06811beb1faca27c936c7232220625fc004ffdb)